packages/block-library/src/navigation/index.php

```diff
@@ -294,34 +231,9 @@ function( $a, $b ) {
  * @return array the normalized parsed blocks.
  */
 function block_core_navigation_get_classic_menu_fallback_blocks( $classic_nav_menu ) {
-	// BEGIN: Code that already exists in wp_nav_menu().
-	$menu_items = wp_get_nav_menu_items( $classic_nav_menu->term_id, array( 'update_post_term_cache' => false ) );
-
-	// Set up the $menu_item variables.
-	_wp_menu_item_classes_by_context( $menu_items );
-
-	$sorted_menu_items = array();
-	foreach ( (array) $menu_items as $menu_item ) {
-		$sorted_menu_items[ $menu_item->menu_order ] = $menu_item;
-	}
-
-	unset( $menu_items, $menu_item );
-
-	// END: Code that already exists in wp_nav_menu().
-
-	$menu_items_by_parent_id = array();
-	foreach ( $sorted_menu_items as $menu_item ) {
-		$menu_items_by_parent_id[ $menu_item->menu_item_parent ][] = $menu_item;
-	}
-
-	$inner_blocks = block_core_navigation_parse_blocks_from_menu_items(
-		isset( $menu_items_by_parent_id[0] )
-			? $menu_items_by_parent_id[0]
-			: array(),
-		$menu_items_by_parent_id
-	);
+	_deprecated_function( __FUNCTION__, '5.8.0', 'WP_Navigation_Gutenberg::get_classic_menu_fallback_blocks' );
 
-	return serialize_blocks( $inner_blocks );
+	return WP_Navigation_Gutenberg::get_classic_menu_fallback_blocks( $classic_nav_menu );
```

Comment:
We should leave all the functions that we deprecate intact so what people who use them don't get surprising results when we chanage things in `WP_Navigation_Gutenberg` in the future.

Reply:
I have now restored the original code but marked all the relevant functions as "deprecated". See `Todos` in the description for further steps which I will make in a follow up.